### ISSUE TITLE
Validate and return url for external cover images in thumbCoverImage()

### ISSUE THREAD

**Issue by dimobelov** [bludit/bludit#807]
https://github.com/bludit/bludit/blob/b4caeb435aba92d921b3dbcbd121d85b6247d702/bl-kernel/pagex.class.php#L308

Simple fix like https://github.com/bludit/bludit/blob/b4caeb435aba92d921b3dbcbd121d85b6247d702/bl-kernel/pagex.class.php#L290
```
	public function thumbCoverImage()
	{
		$coverImageFilename = $this->coverImage(false);
		if ($coverImageFilename==false) {
			return false;
		}
		// Check is external cover image
		if (filter_var($coverImageFilename, FILTER_VALIDATE_URL)) {
			return $coverImageFilename;
		}
		return DOMAIN_UPLOADS_THUMBNAILS.$coverImageFilename;
	}
```

**Comment by dignajar** [comment]
Implemented, thanks

**Comment by dimobelov** [comment]
Push it to git?

**Comment by dignajar** [comment]
Implemented in Bludit v3.1.0. Thanks!
